doc/carddatabase_v4/cards.xsd

```diff
@@ -15,6 +15,7 @@
             <xs:element type="xs:string" name="longname" minOccurs="0" maxOccurs="1" />
             <xs:element type="xs:string" name="settype" minOccurs="0" maxOccurs="1" />
             <xs:element type="xs:string" name="releasedate" minOccurs="0" maxOccurs="1" />
+            <xs:element type="xs:integer" name="priority" minOccurs="1" maxOccurs="1"/>
```

Comment:
If this becomes a required field, we might need to bump the version. I'd rather not do that.